cmake/mpiexec.jet

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
If you can't download the test data to Jet, how did you test this file?

Reply:
I transferred the data from Hera and staged it into the build/tests/*/data directories.

Reply:
Ok. I commented out the ftp of the test data. The file did successfully kick off all the jobs (although some of them failed as expected.) When do you expect ftp access from Jet.

Reply:
I don't know how long these things take, but I submitted a request that was acknowledged on 2/3.  Would you like to be added to the ticket?

Reply:
That is not necessary. Just curious. Once access is granted, we might want to update the build_all.sh script.